controllers/yelp.controller.js

```diff
@@ -9,19 +8,6 @@ var yelp = new Yelp({
   token_secret: keys.yelp.tokenSecret
 });
 
-var defaultSearch = {
-  category_filter: 'restaurants',
-  limit: 5
-};
-
-exports.search = function search(req, res) {
-  var searchParams = _.assign(defaultSearch, req.query);
-
-  yelp.search(searchParams)
-  .then(function returnResults(results) {
-    res.status(200).json(results);
-  })
-  .catch(function catchError(error) {
-    res.status(404).json(error);
-  });
+exports.search = function search(searchParams) {
+  return yelp.search(searchParams);
```

Comment:
Perhaps add a check to ensure valid searchPrarams inputs before sending them on to make the db call?